docs/api-designer/getting-started-api-designer/README.adoc

```diff
@@ -135,40 +134,41 @@ image::create-api-designer-design.png[Image of page to create an API design]
 . Select the design title and click the pen icon. Change the design title to *Orders* and click the *Save changes* icon.
 . Click the *Design* tab, and optionally edit the design as follows:
 .. Provide a version number and a description.
-.. AsyncAPI only: Define terms of service.
+.. _AsyncAPI only:_ Define terms of service.
 .. Add your contact information (name, email address, and URL).
 .. Select a license.
-.. OpenAPI only: Define tags.
+.. _OpenAPI only:_ Define tags.
 .. Define one or more servers.
 .. Configure a security scheme.
-.. OpenAPI only: Specify security requirements.
-.. OpenAPI only: Configure vendor extensions.
+.. _OpenAPI only:_ Specify security requirements.
+.. _OpenAPI only:_ Configure vendor extensions.
 +
 [.screencapture]
 .Editing an API design
```

Comment:
Does this image add value beyond the list of properties you've already described above? Also, given the large size of the screenshot, I found it quite hard to pick out particular details, even with my browser zoomed right in.

Reply:
Removed image